README.md

```diff
@@ -3,6 +3,11 @@
 
 ![Egeria Logo](assets/img/ODPi_Egeria_Logo_color.png)
 
+[![GitHub](https://img.shields.io/github/license/odpi/egeria)](LICENSE.md)
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3044/badge)](https://bestpractices.coreinfrastructure.org/projects/3044)
+[![Azure](https://dev.azure.com/odpi/egeria/_apis/build/status/odpi.egeria)](https://dev.azure.com/odpi/Egeria/_build)
```

Comment:
I wonder if rather than azure we say 'Build' or 'Build pipelines'

We could also add a link to maven artifacts - 'snapshots' https://odpi.jfrog.io/odpi/egeria-snapshot/ & releases at maven central https://mvnrepository.com/artifact/org.odpi.egeria